openerp/http.py

```diff
@@ -1238,11 +1246,10 @@ def assert_valid(self, force=False):
 
         Ensures this session is valid (logged into the openerp server)
         """
-        if self.uid and not force:
+        if self.uid and self.session_token and not force:
```

Comment:
Hello guys, I need your expertize on the modification of this method. I am not sure what are the consequences of this change.

In my knowledge their should never be a session with a login & db without the matching uid, meaning it was doing nothing ??